### PR TITLE
[8.0] TRANSIFEX Support

### PR DESCRIPTION
**Pull request by ivantodorovich**
https://github.com/OCA/crm/issues/82

@max3903 is this enough?

related to https://github.com/OCA/maintainer-quality-tools/issues/194

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,10 +13,16 @@ addons:
...
-  - VERSION="8.0" LINT_CHECK="1"
+  global:
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - TRANSIFEX_USER='transbot@odoo-community.org'
+  - secure: FFyWAwM7PmCbOI8lSIFNkDknkspQQv3sD8aKyz5tTPtAmLdALEj+MaPsuZJS9PoHzqjrylrhHEg+lB5lNjHR+3t2oOgHZb44l+mob22N6WDB5nag11jnBI+q0hGEHrbYNe5r3mnNvUQKXSfmjyKE181755fnOvhc88x5DnHXo0Q=
+
+  matrix:
+  - TRANSIFEX="1"
   - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
   - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
```
Comment (max3903) [inline .travis.yml]: Should be:

```
  - LINT_CHECK="1"
  - TRANSIFEX="1"
  - TESTS="1" ODOO_REPO="odoo/odoo"
  - TESTS="1" ODOO_REPO="OCA/OCB"
```

**Comment by max3903** [comment]
and there is a dash missing before coveralls at the end of the file.

Otherwise :+1:

**Comment by ivantodorovich** [comment]
Done!

Had to amend my last commit because I wasn't using my regular PC, and I forgot to configure git user and email. That's why oca-clabot complained, but I believe it's ok now.

File: .travis.yml [hunk 2]
```diff
@@ -30,4 +37,4 @@ script:
   - travis_run_tests
 
 after_success:
-  coveralls
+  - coveralls
```
Comment (pedrobaeza) [inline .travis.yml]: Please put `- travis_after_tests_success`:

https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml#L78

Comment (ivantodorovich) [inline .travis.yml]: Replacing `coveralls`, right?

**Comment by pedrobaeza** [comment]
:+1: please make the same with 9.0

**Pull request merged**